app/views/organisations/show.html.slim

```diff
@@ -16,4 +16,4 @@
       - tabs.with_navigation_item text: t("organisations.show.tabs.organisation"), link: organisation_path(@organisation)
       - tabs.with_navigation_item text: t("organisations.show.tabs.schools"), link: organisation_schools_path(@organisation)
 
-= render "organisation", preview: false, organisation: @organisation, vacancies: @vacancies
+= render "organisation", preview: false, organisation: @organisation, vacancies: @vacancies, live_group_vacancies_count: @organisation.live_group_vacancies.count
```

Comment:
Suggested to remove the `live_group_vacancies_count: @organisation.live_group_vacancies.count` from here and below call.